LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
```

Comment:
Is this the licence of the flatpak package?

Reply:
It's the intended license of the flatpak package (though we're completely flexible if another license is preferred for it). We're also using MIT as the upstream license on Kolibri itself.